tests/manual-test-cases/Group18-VIC-UI/setup-testbed.robot

```diff
@@ -207,7 +207,7 @@ Deploy Nimbus BrowserVm For NGC Testing
     [Return]  ${user}-${name}  ${ip}
 
 Deploy Nimbus ESXi Server For NGC Testing
```

Comment:
This is just the standard nimbus esxi deploy is it not?  Why not use our existing keyword with the version set to the right one?